### PR TITLE
Better performance with visualization

### PR DESCRIPTION
**Pull request by DaddyWesker**
Working with[ this PR](https://github.com/trueagi-io/Vereya/pull/59) only.

### CODE REVIEW

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 1]
```diff
@@ -83,4 +92,4 @@ def __init__(self, message: TimestampedUnsignedCharVector,
 
     @property
     def pixels(self):
-        return cv2.imdecode(np.frombuffer(self._pixels, dtype="uint8"), cv2.IMREAD_COLOR)
+        return cv2.cvtColor(cv2.flip(np.frombuffer(self._pixels, dtype="uint8").reshape((self.iheight, self.iwidth, self.ich))[:,:,:3],0), cv2.COLOR_RGB2BGR)
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:95]: i think we can use numpy.flip and remove BGR conversion - this way we can drop opencv dependency.

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py:95]: i can't drop rgb2bgr since it is necessary. Numpy flip - ok.

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py]: Done

**Comment by DaddyWesker** [comment]
Ok, this PR is not Draft.

**Review by CICS-Oleg — APPROVED** [review]

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 2]
```diff
@@ -74,13 +76,16 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.zPos = loadedjson['z']
         self.yaw = loadedjson['yaw']
         self.pitch = loadedjson['pitch']
-        self.modelViewMatrix = np.reshape(np.asarray(loadedjson['modelViewMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
+        self.iheight = loadedjson['img_height'][0]
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:79]: I guess it should be the same as pitch, yaw pos
```suggestion
        self.iheight = loadedjson['img_height']
```

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py:79]: Ok, but don't remove [0] please.

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py]: @noskill  Done

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 3]
```diff
@@ -74,13 +76,16 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.zPos = loadedjson['z']
         self.yaw = loadedjson['yaw']
         self.pitch = loadedjson['pitch']
-        self.modelViewMatrix = np.reshape(np.asarray(loadedjson['modelViewMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
+        self.iHeight = loadedjson['img_height'][0]
+        self.iWidth = loadedjson['img_width'][0]
+        self.iCh = loadedjson['img_ch'][0]
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:81]:
```suggestion
        self.iCh = loadedjson['img_ch']
```

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 4]
```diff
@@ -74,13 +76,16 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.zPos = loadedjson['z']
         self.yaw = loadedjson['yaw']
         self.pitch = loadedjson['pitch']
-        self.modelViewMatrix = np.reshape(np.asarray(loadedjson['modelViewMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
+        self.iHeight = loadedjson['img_height'][0]
+        self.iWidth = loadedjson['img_width'][0]
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:80]:
```suggestion
        self.iWidth = loadedjson['img_width']
```

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 5]
```diff
@@ -74,13 +76,16 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.zPos = loadedjson['z']
         self.yaw = loadedjson['yaw']
         self.pitch = loadedjson['pitch']
-        self.modelViewMatrix = np.reshape(np.asarray(loadedjson['modelViewMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
+        self.iHeight = loadedjson['img_height'][0]
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:79]:
```suggestion
        self.iHeight = loadedjson['img_height']
```

**Review by noskill — APPROVED** [review]

**Comment by DaddyWesker** [comment]
@noskill  Done

**Pull request merged**